filetests/call.wat

```diff
@@ -6,5 +6,6 @@
   (func $inc (result i32)
     (i32.const 1)
   )
+  (export "main" (func $main))
   (start $main)
```

Comment:
The start function is called automatically as part of instantiation, so if main is also exported and invoked from the outside, that means it's getting called twice.

Reply:
Hmmm, this "start" function. AFAIK, there has been some discussion around the "start" feature and how it could be used to initialize C++-global-static-constructors. Others assume it is valid only dealing with wasm-apps, not wasm-libs; here "start" would not point to user-defined function, but the start-function genenerated by the compiler/linker. AFAICS, semantics of "start" is not well defined yet.

Reply:
It's unclear what tools will use it for, but it is defined to be [automatically called during instantiation](http://webassembly.github.io/spec/core/bikeshed/index.html#start-function%E2%91%A0).